lib/manager/cocoapods/artifacts.spec.ts

```diff
@@ -10,18 +10,29 @@ import * as _env from '../../util/exec/env';
 import type { StatusResult } from '../../util/git/types';
 import type { UpdateArtifactsConfig } from '../types';
 import { updateArtifacts } from '.';
+import * as fsutil from '../../util/fs';
 
 jest.mock('fs-extra');
 jest.mock('child_process');
 jest.mock('../../util/exec/env');
 jest.mock('../../util/git');
 jest.mock('../../platform');
 jest.mock('../../datasource');
+jest.mock('../../util/fs', () => {
+  const originalModule = jest.requireActual('../../util/fs');
+
+  return {
+    __esModule: true,
+    ...originalModule,
+    readLocalFile: jest.fn(),
+  };
+});
```

Comment:
```suggestion
jest.mock('../../util/fs');
```
Do a full mock and fix tests.